src/request.js

```diff
@@ -0,0 +1,53 @@
+//https://shyr.io/blog/axios-with-react-for-making-requests
+import axios from 'axios'
+
+
+const GRADERX_API = process.env.GRADERX_API
+    ? process.env.GRADERX_API
+    : "http://localhost:5000/";
+
+
+/**
+ * Create an Axios Client with defaults
+ */
+const client = axios.create({
+    baseURL: GRADERX_API
+});
+
+
+
+/**
+ * Request Wrapper with default success/error actions
+ */
+const request = function (options) {
+    const onSuccess = function (response) {
+        console.debug('Request Successful!', response);
+        return response.data;
+    }
+
+    const onError = function (error) {
+        console.error('Request Failed:', error.config);
+
+        if (error.response) {
+            // Request was made but server responded with something
+            // other than 2xx
+            console.error('Status:', error.response.status);
+            console.error('Data:', error.response.data);
+            console.error('Headers:', error.response.headers);
+
+
+        } else {
+            // Something else happened while setting up the request
+            // triggered the error
+            console.error('Error Message:', error.message);
+        }
+
+        return Promise.reject(error.response || error.message);
+    }
+
+    return client(options)
```

Comment:
if you wanted to use the `async`/`await` syntax you'll need `babel` to transpile `es6` otherwise you can leave it as is.